src/components/mobile/mobileAssetView.tsx

```diff
@@ -0,0 +1,302 @@
+        ];
+      } else if (isAssetObject(data) && item === "chassis") {
+        return [
+          <tr>
+            <td key={item}>
+              <p className="label">{AssetFieldsTable[item]}:</p>
+            </td>
+
+            <td style={getChangePlanRowStyle(data)}>
+              {" "}
+              <p>{data.chassis ? data.chassis.hostname : null}</p>
+            </td>
+          </tr>,
+        ];
+      } else if (item === "comment") {
+        dat = <p className="comment">{data[item]}</p>;
+      } else if (isDatacenterObject(data[item])) {
+        dat = <p>{data[item].name}</p>;
+      } else if (!isObject(data[item])) {
+        //TO DO: decide how to render dicts
+        dat = <p>{data[item]}</p>;
+      }
+
+      if (isAssetObject(data)) {
+        return AssetFieldsTable[item] ? (
+          <tr>
+            <td key={item}>
+              <p className="label">{AssetFieldsTable[item]}:</p>
+            </td>
+
+            <td style={getChangePlanRowStyle(data)}>{dat}</td>
+          </tr>
+        ) : null;
+      }
+      if (isModelObject(data)) {
+        const isCustomField =
+          this.props.data_override && this.props.data_override[item];
+        return (
+          <tr>
+            <td key={item}>
+              <p
+                className="label"
+                style={{
+                  fontWeight: isCustomField
+                    ? ("bold" as any)
+                    : ("normal" as any),
+                  color: isCustomField ? "#48AFF0" : "#bfccd6",
+                }}
+              >
+                {ModelFieldsTable[item]}:
+              </p>
+            </td>
+
+            {isCustomField && item !== "display_color" ? (
+              <td>{this.props.data_override[item]}</td>
+            ) : (
+              <td style={getChangePlanRowStyle(data)}>{dat}</td>
+            )}
+          </tr>
+        );
+      } else {
+        return (
+          <tr>
+            <td key={item}>
+              <p className="label">{item}:</p>
+            </td>
+
+            <td>{dat}</td>
+          </tr>
+        );
+      }
+    });
+  }
+
+  getNetworkConnectionForPort(port: string) {
+    return this.props.asset.network_connections.find(
+      (connection: NetworkConnection) => connection.source_port === port
+    );
+  }
+
+  private renderPower() {
+    if (this.props.asset && this.props.asset.model) {
+      if (this.props.asset.model.model_type === MountTypes.BLADE) {
+        return (
+          <p>Blade Power View</p>
+          // <BladePowerView
+          //   {...this.props}
+          //     token={this.props.token}
+          //   asset={this.props.asset}
+          //   shouldUpdate={this.state.powerShouldUpdate}
+          //   updated={() => {
+          //     this.setState({ powerShouldUpdate: false });
+          //   }}
+          //   assetIsDecommissioned={
+          //     this.props.asset.decommissioning_user !== undefined
+          //   }
+          // />
```

Comment:
Currently commented out because I have not updated BladePowerView to fit mobile requirements, but will do in next PR